src/js/common/schemaform/FormApp.jsx

```diff
@@ -1,221 +1,55 @@
-      // Chrome requires this to be set
-      e.returnValue = message;     // eslint-disable-line no-param-reassign
-    }
-    return message;
-  }
-
-  redirectOrLoad(props) {
-    // Stop a user that's been redirected to be redirected again after logging in
-    this.shouldRedirectOrLoad = false;
-
-
-    const firstPagePath = props.routes[props.routes.length - 1].pageList[0].path;
-    // If we're logged in and have a saved / pre-filled form, load that
-    if (props.isLoggedIn) {
-      const currentForm = props.formConfig.formId;
-      const isSaved = props.savedForms.some((savedForm) => savedForm.form === currentForm);
-      const hasPrefillData = props.prefillsAvailable.includes(currentForm);
-      const saveEnabled = !this.props.formConfig.disableSave;
-      if (saveEnabled && (isSaved || hasPrefillData)) {
-        props.fetchInProgressForm(currentForm, props.formConfig.migrations, !isSaved && hasPrefillData);
-      } else {
-        // No forms to load; go to the beginning
-        // If the first page is not the intro and uses `depends`, this will probably break
-        props.router.replace(firstPagePath);
-      }
-    } else {
-      // Can't load a form; go to the beginning
-      // If the first page is not the intro and uses `depends`, this will probably break
-      props.router.replace(firstPagePath);
-    }
-  }
-
-  removeOnbeforeunload = () => {
-    window.removeEventListener('beforeunload', this.onbeforeunload);
-  }
-
   render() {
-    const { currentLocation, formConfig, children, formData, loadedStatus } = this.props;
+    const { currentLocation, formConfig, children, formData } = this.props;
     const trimmedPathname = currentLocation.pathname.replace(/\/$/, '');
     const isIntroductionPage = trimmedPathname.endsWith('introduction');
     const isConfirmationPage = trimmedPathname.endsWith('confirmation');
     const GetFormHelp = formConfig.getHelp;
-    let content;
-    const loadingForm = trimmedPathname.endsWith('resume') || loadedStatus === LOAD_STATUSES.pending;
-    if (!formConfig.disableSave && loadingForm) {
-      content = <LoadingIndicator message="Retrieving your saved form..."/>;
-    } else if (!formConfig.disableSave && this.props.savedStatus === SAVE_STATUSES.pending) {
-      content = <LoadingIndicator message="Saving your form..."/>;
-    } else if (!formConfig.disableSave && this.shouldRedirectOrLoad) {
-      content = <LoadingIndicator message="Retrieving your profile information..."/>;
-    } else if (!isInProgress(trimmedPathname)) {
-      content = children;
-    } else {
-      content = (
-        <div>
-          <FormNav formData={formData} formConfig={formConfig} currentPath={trimmedPathname}/>
-          <div className="progress-box progress-box-schemaform">
-            {children}
-          </div>
-        </div>
-      );
+    const saveEnabled = !formConfig.disableSave;
+
+    let formTitle;
+    let formNav;
+    if (!isIntroductionPage) {
```

Comment:
Consulted with @goldenmeanie on the difference between how the title is rendered on the intro page vs. every other page and he's going to check with content folks to see if it needs to be different. May keep this as a separate ToDo.